.github/CODEOWNERS

```diff
@@ -33,7 +33,7 @@
 /src/azure-cli/azure/cli/command_modules/privatedns/ @jsntcy @haroldrandom @kairu-ms @msyyc
 /src/azure-cli/azure/cli/command_modules/policyinsights/ @cheggert
 /src/azure-cli/azure/cli/command_modules/profile/ @jiasli @evelyn-ys @fengzhou-msft
-/src/azure-cli/azure/cli/command_modules/rdbms/ @Juliehzl @evelyn-ys
+/src/azure-cli/azure/cli/command_modules/rdbms/ @arde0708 @Juliehzl @evelyn-ys
```

Comment:
👍 Good to have a member from service team to ensure the code quality.

Reply:
You may link to the original issue/PR to make the purpose of this PR clearer.

Reply:
@evelyn-ys nice!